### DISCUSSION TITLE
关于付费收款方式

### DISCUSSION THREAD

**Discussion by ccw2018** [dashingsoft/pyarmor#1986] [Q&A]
我司是研究所，采购软件，需要发票，并且收款单位需要和发票开具单位一样，可否提供对公账户

**Comment by jondy** [comment]
抱歉，目前不提供对公帐户

**Reply by ccw2018** [reply]
可否去办理下，对于研究所这类机构非常需要对公账户，我们非常支持pyarmor这个技术架构

**Comment by jondy** [comment]
感谢你的认可，但是 Pyarmor 就是一个小工具，暂时没有这方面的计划
